mep/common/migrations/0003_content_editor_perms_location.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.9 on 2018-02-08 20:16
+from __future__ import unicode_literals
+
+from django.contrib.auth.management import create_permissions
+from django.db import migrations
+
+
+new_content_editor_perms = [
+    'add_address', 'change_address', 'delete_address',
+    'add_location', 'change_location', 'delete_location',
+]
+
+
+def update_content_editor_group(apps, schema_editor):
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+
+    # make sure permissions are created before loading the fixture
+    # which references them
+    # (when running migrations all at once, permissions may not yet exist)
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True
+        create_permissions(app_config, apps=apps, verbosity=0)
+        app_config.models_module = None
+
+    editor_group = Group.objects.get(name='Content Editor')
+    permissions = []
+    for codename in new_content_editor_perms:
+        # using explicit get so that there will be an error if an
+        # expected permission is not found
+        permissions.append(Permission.objects.get(codename=codename))
```

Comment:
This appears to be an offending line when running this permissions migration on a dump of production as of today. Stack trace is below in hopes it might be more revealing to you, @rlskoeser, but it's an issue of the Permission having a duplicate in the old scheme, I think:

```
  File "./manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/core/management/commands/migrate.py", line 204, in handle
    fake_initial=fake_initial,
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/db/migrations/executor.py", line 115, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/db/migrations/executor.py", line 145, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/db/migrations/executor.py", line 244, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/db/migrations/migration.py", line 126, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/db/migrations/operations/special.py", line 193, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/Users/bhicks/repos/mep-django/mep/common/migrations/0003_content_editor_perms_location.py", line 32, in update_content_editor_group
    permissions.append(Permission.objects.get(codename=codename))
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/bhicks/repos/mep-django/env/lib/python3.6/site-packages/django/db/models/query.py", line 384, in get
    (self.model._meta.object_name, num)
__fake__.MultipleObjectsReturned: get() returned more than one Permission -- it returned 2!
```